src/components/gui/gui.css

```diff
@@ -99,6 +99,7 @@
 .blocks-wrapper {
     flex-grow: 1;
     position: relative;
+    z-index: 0;
```

Comment:
What does this do?

Reply:
My button was ending up behind the flyout, so it was faded out and unclickable. This probably isn't the right solution... thoughts?

Reply:
I think this makes sense for that. It might be clearer if you put `z-index: 1` here and `z-index: 2` on the button though (plus a comment).

Reply:
I'm not sure how nested z-indices work, but the vertical category menu will put a 100% height div at z-index of 40 or so, it might make more sense to just z-index the button above that?